.github/workflows/main.yml

```diff
@@ -11,11 +11,12 @@ permissions:
 
 jobs:
   test:
-    name: Test - Python ${{ matrix.python-version }}
-    runs-on: ubuntu-latest
+    name: Test - Python ${{ matrix.python-version }} on ${{ matrix.os }}
```

Comment:
Now:
<img width="264" alt="image" src="https://github.com/user-attachments/assets/926789df-47ff-4a83-b079-c85a99fd1c80" />


I usually omit this:
```suggestion
```

The default is quite concise:

<img width="256" alt="image" src="https://github.com/user-attachments/assets/ba572d54-a2eb-4eb4-8c32-6fa9aed4e2d7" />

https://github.com/termcolor/termcolor/actions/runs/13591158653